.github/workflows/release.yml

```diff
@@ -149,3 +149,6 @@ jobs:
         foreach ($example in $examples) { dotnet build $example.FullName }
       if: runner.os == 'Windows'
       shell: powershell
+    - name: 🧪 Test Templates
+      run: |
+        uses: ./.github/actions/test-templates
```

Comment:
We need to fix test-templates, it currently install the templates from the current build. Here we want to use the package we download from the release artifacts. Maybe the simpler is to move the install to a separate step for each use case.

Reply:
Done, I just removed the step from the `test-templates` that installs it and added a separate step in both `ci.yml` and `release.yml` that installs the templates there!